Doc/api/conf.py

```diff
@@ -82,7 +82,7 @@
 # -- Options for autodoc --------------------------------------------------
 
 # This requires Sphinx 1.8.0b1 or later:
-autodoc_default_values = {
+autodoc_default_options = {
```

Comment:
Can you update the comment above to say v1.8 or later please?

(It seems only some of the beta used ``autodoc_default_values``)

Reply:
How's this?